### ISSUE TITLE
What is Git?

### ISSUE THREAD

**Issue by schalkneethling** [The-Mycelium-Network/webworkbook#21]

**Comment by samyaksand** [comment]
Hello, Im interested in working on this. Please assign me this issue.

**Comment by schalkneethling** [comment]
@CoyaG, some things to keep in mind:

When writing content, try to focus on writing a synopsis and linking to the content for further reading.
For videos, feel free to use Hugo shortcodes to embed the video in context.

https://gohugo.io/content-management/shortcodes/#use-hugos-built-in-shortcodes

For this one, you would create a new folder called `what-is-git` in https://github.com/The-Mycelium-Network/webworkbook/tree/main/content/topics and then an `index.md` inside that folder. Refer to the existing pages for the frontmatter that needs to be added to a page, for example:

```markdown
---
title: What is the internet? - web://workbook
keywords: what is the internet, internet explained, internet 101
description: A collection of informational pieces that explains what the internet is.
menu:
  main:
    parent: topics
    name: What is the internet?
---
```

**Comment by bpires** [comment]
**Interesting stats to include somewhere on the Git page:**

[Stack Overflow Developer Survey 2022](https://survey.stackoverflow.co/2022/#section-version-control-version-control-systems) states that no other technology is as widely used as Git, with 97% of professional developers pointing it as their primary version control system. The survey also shows that the command line is the primary way developers interact with their version control system.

**Comment by CoyaG** [comment]
Holy moly that is interesting! Has anyone added that yet?

**Comment by bpires** [comment]
> Holy moly that is interesting! Has anyone added that yet?

No, you'd be the perfect person to do it @CoyaG 😅  As you are the author of PR #42, you can add it to the `What is Git?` text content, where it seems fit, by just pushing a new commit to your `webworkbook-coyag-branch-2` branch on your fork. The new commit will be added to the same PR. 

Try it! Just remember to update your fork first and then do a `git pull` on your local environment, so you get the most updated version of your branch before starting to work on it.

**Comment by bpires** [comment]
Some articles about Git from freeCodeCamp:

- [Git Cheat Sheet – 50 Git Commands You Should Know](https://www.freecodecamp.org/news/git-cheat-sheet/)
- [How to Use Git and GitHub – Version Control Basics for Beginners](https://www.freecodecamp.org/news/git-and-github-the-basics/)
- [Basic Git Commands – How to Use Git in a Real Project](https://www.freecodecamp.org/news/how-to-use-basic-git-and-github-commands/)
- [Git Best Practices – How to Write Meaningful Commits, Effective Pull Requests, and Code Reviews](https://www.freecodecamp.org/news/git-best-practices-commits-and-code-reviews/)
- [How to Use Git and GitHub – Introduction for Beginners](https://www.freecodecamp.org/news/introduction-to-git-and-github/)
- [How to Fork a GitHub Repository – A Complete Workflow](https://www.freecodecamp.org/news/how-to-fork-a-github-repository/)
- [Git and GitHub Tutorial – Version Control for Beginners](https://www.freecodecamp.org/news/git-and-github-for-beginners/)
- [Git for Professionals – Free Version Control Course](https://www.freecodecamp.org/news/git-for-professionals/)

I'll review them later to curate the best ones, but decided to leave them here for further reference. If any caught your eye, feel free to leave your opinion about it and about if we should include it among the webworkbook's resources.

**Comment by CoyaG** [comment]
* 50 Git Commands is already in the doc
* How to use Git & GitHub-Version Control Basics for Beginners: this one seems kind of similar to "How to use Git and GitHub-Intro for Beginners" I feel as though the latter (Intro for Beginners) may be more in detail and provides links to other useful beginner resources.

Everything else looks cool : 😄
